android/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.appboy.reactbridge">
 
-  <application android:allowBackup="true"
+  <application android:allowBackup="false"
```

Comment:
@SMJ93 feel free to remove this attribute altogether, braze has no interest in specifying it